xrpl/models/transactions/transaction.py

```diff
@@ -69,7 +75,7 @@ def _key_to_tx_json(key: str) -> str:
 def _value_to_tx_json(value: XRPL_VALUE_TYPE) -> XRPL_VALUE_TYPE:
     # IssuedCurrencyAmount and PathStep are special cases and should not be snake cased
     # and only contain primitive members
```

Comment:
Should this comment live with the _LOWER_CASE_MODELS object now so it doesn't fall out of date?